vars/mtaBuild.groovy

```diff
@@ -28,6 +31,16 @@ def call(Map parameters = [:]) {
                                       parameters, parameterKeys,
                                       stepConfiguration, stepConfigurationKeys, stepDefaults)
 
+
+        MTA_JAR_FILE_VALIDATE: {
+            // same order like inside getMtaJar,
+            def mtaJarLocation = configuration?.mtaJarLocation ?: env?.MTA_JAR_LOCATION
+            def returnCodeLsMtaJar = sh script: "ls ${DEFAULT_MTA_JAR_NAME}", returnStatus:true
+            if(mtaJarLocation || ( !mtaJarLocation && returnCodeLsMtaJar != 0)) {
+                toolValidate tool: 'mta', home: configuration?.mtaJarLocation ?: env?.MTA_JAR_LOCATION
```

Comment:
use `mtaJarLocation` from above

Reply:
not understood, explain.

Reply:
Ah, ok we have the mtaJarLocation in link 37, which should of course be re-used. I will change that.

Reply:
done